### DISCUSSION TITLE
BinSpreader official release date

### DISCUSSION THREAD

**Discussion by jdwinkler-lanzatech** [ablab/spades#930] [General]
Hi,

Thank you for your hard work and dedication in developing the SPAdes family of tools. I am very interested in your latest tool BinSPreader as it seems to be a significant improvement over the current state of the art, especially other graph-based bin refiners. Just for my own planning, do you have a feeling when you will make an official release? I figure there will be refinements between the current  pre publication release and the final product.

**Comment by asl** [comment]
Hello

Thank you for your interest in BinSPreader.

So far we do not have any ETA on the final release date. Likely this would coincide with the final publication, however, I cannot promise for sure, when this would happen :)

So far the most changes we're still planning are around read extraction. So, other features will change only if some critical bugs will be found there.

**Comment by jdwinkler-lanzatech** [comment]
Thank you! I'll keep an eye out for the publication.

**Comment by SilasK** [comment]
The publication is out. 

Did you make a comparison to GraphBin2?

**Reply by asl** [reply]
The paper was published ~year ago. It contained the comparison with MetaCoAg which is a successor of GraphBin2.

Here is a summary table with mean F1 scores across tools and samples

![image](https://github.com/ablab/spades/assets/67392/210d4fa4-13e5-44e6-80d9-c9263e6860c4)

and here is # of recovered high-quality genomes

![image](https://github.com/ablab/spades/assets/67392/052e9e9d-2050-469a-8c4b-cf38caf703c4)

**Reply by SilasK** [reply]
Thank you for putting the image and telling me the change of the name. apparently, metaCoAG is now more a stand alone binner rather than improves the binning of others.

I think it's difficult to judge based on n HQ bins, because the advantage of binspreader is precisely to get contigs that are not necessarily contributing to the completeness estimated by checkM (more variable part), don't you think?

**Reply by asl** [reply]
> I think it's difficult to judge based on n HQ bins, because the advantage of binspreader is precisely to get contigs that are not necessarily contributing to the completeness estimated by checkM (more variable part), don't you think?

Right. However, it's quite non-trivial to estimate things without knowing the references. Still, there are plenty of supplementary tables that shows the effect of refining on different angles. Let me show some.

<img width="700" alt="Screenshot 2023-07-10 at 21 23 25" src="https://github.com/ablab/spades/assets/67392/6e220a84-7e6a-46da-9601-8fd1e2178f10">

<img width="681" alt="Screenshot 2023-07-10 at 21 24 52" src="https://github.com/ablab/spades/assets/67392/6d72d93e-5cbf-4e0e-947b-d0979d67c03f">

<img width="693" alt="Screenshot 2023-07-10 at 21 25 21" src="https://github.com/ablab/spades/assets/67392/bd7662a5-c61a-461d-a64a-73c1da0df8ff">

<img width="657" alt="Screenshot 2023-07-10 at 21 26 02" src="https://github.com/ablab/spades/assets/67392/dba94c75-bb75-4b2c-b6ba-96994c224606">
